src/components/KEmptyState/KEmptyState.vue

```diff
@@ -12,7 +12,7 @@
         <KIcon
           :color="isError ? iconColor || 'var(--black-70)' : iconColor"
           :icon="icon ? icon : 'warning'"
-          :secondary-color="isError ? 'var(--yellow-400)' : undefined"
+          :secondary-color="isError ? iconSecondaryColor || 'var(--yellow-400)' : iconSecondaryColor"
```

Comment:
can you provide the fallback as well
```suggestion
          :secondary-color="isError ? iconSecondaryColor || 'var(--yellow-400, #fabe5f)' : iconSecondaryColor"
```